public/app/features/dashboard-scene/utils/utils.ts

```diff
@@ -164,6 +165,11 @@ export function getQueryRunnerFor(sceneObject: SceneObject | undefined): SceneQu
 
 export function getDashboardSceneFor(sceneObject: SceneObject): DashboardScene {
   const root = sceneObject.getRoot();
+
+  if (root instanceof PanelEditor) {
+    return root.state.dashboardRef.resolve();
+  }
```

Comment:
@kaydelaney  - I've pushed this update to make panel edit work-  i had erorrs when navigating to panel edit.